flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -27,22 +29,70 @@
     //  general checkpoint and state backend options
     // ------------------------------------------------------------------------
 
-    /** The state backend to be used to store and checkpoint state. */
+    /**
+     * The checkpoint storage used to store state.
+     *
+     * <p>The implementation can be specified either via their shortcut name, or via the class name
+     * of a {@code StateBackendFactory}. If a StateBackendFactory class name is specified, the
+     * factory is instantiated (via its zero-argument constructor) and its {@code
+     * StateBackendFactory#createFromConfig(ReadableConfig, ClassLoader)} method is called.
+     *
+     * <p>Recognized shortcut names are 'hashmap' and 'rocksdb'.
+     */
```

Comment:
I guess this change and the one below were made by mistake. This is actually about `CHECKPOINT_STORAGE`, right?

Reply:
I've expanded on the javadoc but this is about the statebackend.